applications/mlflow/charts/mlflow/examples/database/external/existing-secret.yaml

```
@@ -0,0 +1,19 @@
+# MLFlow with external PostgreSQL configuration using existing secret
+
+# PostgreSQL configuration
+postgres:
+  # Embedded PostgreSQL is disabled
+  embedded:
+    enabled: false
+
+  # External PostgreSQL is enabled
+  external:
+    enabled: true
+    host: "your-postgres-host"
+    port: 5432
+    database: mlflow
+
+mlflow:
+  backendStore:
+    # Use an existing secret containing the database connection string
+    existingSecret: "mlflow-postgres-secret" 
```

needs newline at EOF
```suggestion
    existingSecret: "mlflow-postgres-secret"

```